tidb-cloud/size-your-cluster.md

```diff
@@ -344,3 +349,58 @@ You can click any of the following scales to check its performance data.
     | Update Non-index  | 6,000   | 128,601 | 128,601 | 24            |
 
 </details>
+
+### 2 vCPU performance reference
+
+Currently, the 2 vCPU support of TiDB and TiKV is still in beta.
+
+Test environment:
+
+- TiDB version: v6.1.0
+- Warehouses: 1000
+- Data size: 80 G
+- Table size: 10000000
+- Table count: 32
+
+You can check its performance data as follows:
+
+<details>
+<summary>TiDB: 2 vCPU * 2; TiKV: 2 vCPU * 3</summary>
```

Comment:
```suggestion
<summary>TiDB: 2 vCPU \* 2; TiKV: 2 vCPU \* 3</summary>
```

Reply:
Taking L149 as an example, \  is not necessary.